test/utils.py

```diff
@@ -158,7 +158,7 @@ def test_should_not_validate_etag_when_set_as_false(self):
         self.mock_ov_client.connection.disable_etag_validation.assert_called_once()
 
 
-class ParamsTestCase(PreloadedMocksBaseTestCase):
+class FilterParamsTestCase(PreloadedMocksBaseTestCase):
```

Comment:
In these test methods, you are testing all those attributes and not only the filter.